### ISSUE TITLE
`forge` does't work on (my) OSX Sierra

### ISSUE THREAD

**Issue by fbehrens** [fsharp-editing/Forge#14]
## My Environment
OSX Sierra
```
> brew info forge                                                                             master [6e0fed4]
samritchie/forge/forge: stable 1.2.0
Forge (Command line tool for creating F# Projects)
https://forge.run
/usr/local/Cellar/forge/1.2.0 (7 files, 2.8M) *
  Built from source on 2016-11-29 at 21:39:26
From: https://github.com/samritchie/homebrew-forge/blob/master/forge.rb
==> Dependencies
Required: mono ✔

>mono --version                                                                              master [6e0fed4]
Mono JIT compiler version 4.6.2 (Stable 4.6.2.7/08fd525 Tue Nov 22 22:06:40 GMT 2016)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug
	LLVM:          supported, not enabled.
	GC:            sgen
```
When I run forge I got the following errror
```
> forge -h                                                                                    master [6e0fed4]
Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly or one of its dependencies.
File name: 'Nett, Version=0.5.0.0, Culture=neutral, PublicKeyToken=null'
  at Forge.Alias.load () [0x00010] in <5839973d73406ffea74503833d973958>:0
  at Forge.Commands.applyAlias (System.String[] args) [0x00001] in <5839973d73406ffea74503833d973958>:0
  at Forge.Commands.strikeForge (System.String[] args, Forge.Commands+Result cont) [0x00001] in <5839973d73406ffea74503833d973958>:0
  at Forge.App.main (System.String[] argv) [0x000b5] in <5839973d73406ffea74503833d973958>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly or one of its dependencies.
File name: 'Nett, Version=0.5.0.0, Culture=neutral, PublicKeyToken=null'
  at Forge.Alias.load () [0x00010] in <5839973d73406ffea74503833d973958>:0
  at Forge.Commands.applyAlias (System.String[] args) [0x00001] in <5839973d73406ffea74503833d973958>:0
  at Forge.Commands.strikeForge (System.String[] args, Forge.Commands+Result cont) [0x00001] in <5839973d73406ffea74503833d973958>:0
  at Forge.App.main (System.String[] argv) [0x000b5] in <5839973d73406ffea74503833d973958>:0
```

the same error I get with the `Ionide` bundles version
```
> mono ~/.vscode/extensions/Ionide.Ionide-fsharp-2.11.1/bin_forge/Forge.exe -h                master [6e0fed4]

Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly or one of its dependencies.
File name: 'Nett, Version=0.5.0.0, Culture=neutral, PublicKeyToken=null'
  at Forge.Alias.load () [0x00010] in <58399aa373406ffea7450383a39a3958>:0
  at Forge.Commands.applyAlias (System.String[] args) [0x00001] in <58399aa373406ffea7450383a39a3958>:0
  at Forge.Commands.strikeForge (System.String[] args, Forge.Commands+Result cont) [0x00001] in <58399aa373406ffea7450383a39a3958>:0
  at Forge.App.main (System.String[] argv) [0x000b5] in <58399aa373406ffea7450383a39a3958>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly or one of its dependencies.
File name: 'Nett, Version=0.5.0.0, Culture=neutral, PublicKeyToken=null'
  at Forge.Alias.load () [0x00010] in <58399aa373406ffea7450383a39a3958>:0
  at Forge.Commands.applyAlias (System.String[] args) [0x00001] in <58399aa373406ffea7450383a39a3958>:0
  at Forge.Commands.strikeForge (System.String[] args, Forge.Commands+Result cont) [0x00001] in <58399aa373406ffea7450383a39a3958>:0
  at Forge.App.main (System.String[] argv) [0x000b5] in <58399aa373406ffea7450383a39a3958>:0
```

**Comment by Krzysztof-Cieslak** [comment]
Check latest release on GitHub and update your Ionide. 

Already fixed in #13

**Comment by fbehrens** [comment]
Thanks, for quick fix of `ionide`
